fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustGauge.tsx

```diff
@@ -96,26 +96,27 @@ export class ThrustGauge extends DisplayComponent<ThrustGaugeProps> {
 
   private readonly throttleTarget = MappedSubject.create(
     ([throttlePositionN1, thrustLimitIdle, thrustLimitMax, thrIdleOffset]) =>
```

Comment:
To avoid confusion, keep the param names consistent with the input subscribables (you will have to change it in the body of the function to; github does not let me do a suggestion spanning a deleted line).
```suggestion
    ([throttlePositionN1, thrustLimitIdle, thrustLimitToga, thrIdleOffset]) =>
```

Please also apply for the other 2 instances below.